lib/hz2600/Kazoo/Api/Entity/SystemConfig.php

```diff
@@ -1,7 +1,41 @@
 <?php
 
 namespace Kazoo\Api\Entity;
+use \Kazoo\Common\Exception\InvalidArgument;
 
 class SystemConfig extends AbstractEntity
 {
+
+	/**
+	 * perform a change in system config
+	 * $arg can be stdClass or key or array of key, values
+	 * $value is used when $arg is $key
+	 * $key used when $value is not null can be a string or an array 
+	 *
+	 */	
+	public function change($arg, $value = null) {
+		$this->reset(true);
+		if(!$value) {
+			if($arg instanceof \stdClass) {
+				$id = $this->getId();				
+				$this->setEntity($arg);
+				$this->setId($id);				
+			} else {
+				if(!is_array($arg))
```

Comment:
to follow the project style, we should always enclose conditions in curly braces.